src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,161 @@
+  }
+
+  const onFormReset = () => {
+    reset(convertMovieToFormData(movie));
+  }
+
+  return (
+      <form className="movie-form" onSubmit={handleSubmit(onFormSubmit)} autoComplete={'off'}>
+        <div className="row mb-3">
+          <div className="col-7">
+            <label htmlFor="title">Title</label>
+            <input {...register("title", {required: "Title is required"})} id="title"/>
+            <ErrorMessage errors={errors} name="title" render={({ message }) => <p className="error-message">{message}</p>}/>
+          </div>
+
+          <div className="col">
+            <label htmlFor="releaseDate">Release Date</label>
+            <Controller
+                control={control}
+                name="releaseDate"
+                render={({field: {onChange, value}}) => (
+                    <DatePicker id="releaseDate"
+                                selected={value}
+                                placeholderText="Select Date"
+                                onChange={onChange}
+                                dateFormat='yyyy-MM-dd'/>
+                )}
+            />
+          </div>
+        </div>
+
+        <div className="row mb-3">
+          <div className="col-7">
+            <label htmlFor="posterUrl">Movie Url</label>
+            <input {...register("posterUrl")} id="posterUrl" placeholder="https://"/>
+          </div>
+
+          <div className="col">
+            <label htmlFor="rating">Rating</label>
+            <input {...register("rating")} id="rating" type="number" min={0} max={10} step=".1"/>
+          </div>
+        </div>
+
+        <div className="row mb-3">
+          <div className="col-7">
+            <label htmlFor="genres_input">Genre</label>
+            <Controller
+                control={control}
+                name="genres"
+                render={({field: {onChange, value}}) => (
+                    <Multiselect
+                        id="genres"
+                        options={genres}
+                        isObject={false}
+                        placeholder="Select Genre"
+                        onSelect={onChange}
+                        onRemove={onChange}
+                        selectedValues={value}
+                        closeOnSelect={false}
+                    />
+                )}
+            />
+          </div>
+
+          <div className="col">
+            <label htmlFor="duration">Duration</label>
+            <input {...register("duration")} id="duration" type="number" min={0} placeholder="minutes"/>
+          </div>
+        </div>
+
+        <div className="row mb-4">
+          <div className="col">
+            <label htmlFor="overview">Overview</label>
+            <textarea {...register("overview")} id="overview" rows={4} placeholder="Movie Description"/>
+          </div>
+        </div>
+
+        <div className="d-flex justify-content-end">
+          <Button label="Reset" className="mx-2" onClick={onFormReset}></Button>
```

Comment:
Just as a reminder: components without children can be written using self-closing tags.

Reply:
Removed the closing tag